hivemind/p2p/p2p_daemon.py

```diff
@@ -28,13 +28,12 @@ class P2PInterruptedError(Exception):
     pass
 
 
-@dataclass(frozen=False)
+@dataclass(frozen=True)
 class P2PContext(object):
```

Comment:
Naming in `P2PContext` was misleading: before this change, `id` and `port` denoted the server (local) credentials, `peer_id` and `peer_addr` denoted the client (remote) credentials.

This change fixes the naming and removes the local port from the context (the p2p handlers should only operate with PeerIDs and Multiaddrs, the port is too internal concept for them).